katsdptelstate/memory.py

```diff
@@ -110,6 +110,7 @@ def __init__(self, data):
         super(_Callback, self).__init__(string_escape=None)
         self.data = data
         self.n_keys = 0
+        self.client_busy = False
```

Comment:
In theory any of the functions could raise MemoryError. But it's probably not worth worrying about.

Reply:
I agree.